pkg/pipeline/templates/crd_types.go.tmpl

```diff
@@ -38,7 +38,9 @@ type {{ .CRD.Kind }}Status struct {
 
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
+{{ if eq .CRD.MarkStorageVersion "true" -}}
 // +kubebuilder:storageversion
+{{ end }}
```

Comment:
Question: Would `{{- end }}` be more appropriate here? Otherwise, it seems to me that double blank lines will follow.